python_modules/dagster/dagster/_core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -212,13 +212,9 @@ def _connect(self, shard: str) -> Iterator[Connection]:
                 self._initdb(engine)
                 self._initialized_dbs.add(shard)
 
-            conn = engine.connect()
-
-            try:
-                yield conn
-            finally:
-                conn.close()
-            engine.dispose()
```

Comment:
just confirming this is intentionally removed

Reply:
I'll add this back to be safe, but I'm pretty sure this is a no-op because we're using nullpool for the engine.